opentech/apply/dashboard/views.py

```diff
@@ -25,42 +25,41 @@
     PaymentRequestsDashboardTable,
     ProjectsDashboardTable
 )
-from opentech.apply.utils.views import ViewDispatcher
+from opentech.apply.utils.views import RedirectToListViewMixin, ViewDispatcher
 
 
-class AdminDashboardView(TemplateView):
-
-    def get(self, request, *args, **kwargs):
-        # redirect to submissions list when we use the filter to search for something
-        if len(request.GET):
-            query_str = '?'
-            for key, value in request.GET.items():
-                query_str += key + '=' + value + '&'
-            return HttpResponseRedirect(reverse_lazy('funds:submissions:list') + query_str)
```

Comment:
You've broken the search and filter aspects of the table with the current implementation.

TBH, i think this whole view is a bit of a mess (historic reasons)  and could do with some tidying up. 

associating filtersets with tables and prefixes i think would help this out. rather than just dumping everything into the context

Reply:
I missed those on the submissions table, should they redirect to the submissions page?  Or should we remove them from the dashboard and push users to use the Submissions List page instead?